orca/orca-queue/orca-queue.gradle

```diff
@@ -44,4 +44,5 @@ dependencies {
   testImplementation(project(":orca-echo"))
   testImplementation("org.apache.groovy:groovy")
   testImplementation("org.assertj:assertj-core")
+  testImplementation("com.squareup.retrofit2:converter-jackson")
```

Comment:
Don't need this since we've reverted the other changes to orca-queue.  Gonna leave it since those changes are coming soon enough...